nodeadm/api/v1alpha1/nodeconfig_types.go

```diff
@@ -30,6 +31,7 @@ type NodeConfigList struct {
 
 type NodeConfigSpec struct {
 	Cluster      ClusterDetails  `json:"cluster,omitempty"`
+	Kubelet      KubeletOptions  `json:"kubelet,omitempty"`
 	FeatureGates map[string]bool `json:"featureGates,omitempty"`
```

Comment:
Are feature gates scoped to our API, or is this kubelet feature gates?

Reply:
yep its ours not kubelet's; to opt-in for a particular "preset" of the options

Reply:
Is there an example of how to configure this, or is it something we think well use in the future. If the latter, I'd recommend only introducing it when we need it. Sounds like the kind of thing that's design worthy in its own right.